configure.ac

```diff
@@ -7789,6 +7789,73 @@ PY_STDLIB_MOD([_sha2], [test "$with_builtin_sha2" = yes])
 PY_STDLIB_MOD([_sha3], [test "$with_builtin_sha3" = yes])
 PY_STDLIB_MOD([_blake2], [test "$with_builtin_blake2" = yes])
 
+dnl PY_SIMD_DETECT(INSTRUCTION_SET_NAME, COMPILER_FLAG, NORMALIZED_NAME)
+AC_DEFUN([PY_SIMD_DETECT], [
+    AS_VAR_PUSHDEF([py_var], [m4_ifblank([$3],
+        [[ac_cv_can_compile_simd_]m4_tolower([$1])],
+        [[ac_cv_can_compile_simd_]m4_tolower([$3])])])
+    AS_VAR_PUSHDEF([py_define], [m4_ifblank([$3],
+        [[CAN_COMPILE_SIMD_]m4_toupper([$1])[_INSTRUCTIONS]],
+        [[CAN_COMPILE_SIMD_]m4_toupper([$3])[_INSTRUCTIONS]])])
```

Comment:
```suggestion
        [[Py_CAN_COMPILE_SIMD_]m4_toupper([$1])[_INSTRUCTIONS]],
        [[Py_CAN_COMPILE_SIMD_]m4_toupper([$3])[_INSTRUCTIONS]])])
```

Reply:
Alternatively:

```suggestion
        [[Py_HAVE_SIMD_]m4_toupper([$1])[_INSTRUCTIONS]],
        [[Py_HAVE_SIMD_]m4_toupper([$3])[_INSTRUCTIONS]])])
```

Reply:
I think I'll keep the `CAN_COMPILE` because I'd like to keep the name `HAVE_SIMD` for future usage. The macro detected in autoconf is, for now, really just for the compiler and not for the runtime.